go-jni/string.go

```diff
@@ -0,0 +1,39 @@
+package cpg
+
```

Comment:
The purpose of this file seems to be to create constants of different type, can we rename this file to constants.go?